src/System.Runtime.Extensions/src/System/Runtime/CompilerServices/SwitchExpressionException.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Indicates that a switch expression that was non-exhaustive failed to match its input
+    /// at runtime, e.g. in the C# 8 expression <code>3 switch { 4 => 5 }</code>.
+    /// The exception optionally contains an object representing the unmatched value.
+    /// </summary>
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
I don't think you need TypeForwardedFrom since, well , it wasn't. You just made this new type up 😃

Reply:
Unless we expect to port to .NET Framework, in which case you should leave it. @gafter ?

Reply:
Yeah please remove the TypeForwardedFrom here.

Reply:
@danmosemsft as you mentioned that in the standup, I think we should not keep the attribute without knowing that the type will be available in .NET Framework. I would defer adding it until the type is ported over (which I doubt will be).

Reply:
Why would .NET Framework matter? AFAIK we don't support cross-framework serialization (i.e. serialize on .NET Core and deserialize on .NET Framework).